tests/unit/core/test_file_utils.py

```diff
@@ -34,38 +32,11 @@ def test_generate_files_from_paths(
 
     file = files[0]
     assert file.filename == str(path)
-    assert file.document == expected_content
+    assert file.content == expected_content
 
     assert len(files) == 1
 
 
-@pytest.mark.parametrize(
-    ["filename", "creator"],
-    [
-        ("empty_file", lambda x: x.write_text("")),
```

Comment:
We won't skip empty files anymore, is this expected ?

Reply:
We still skip them, but later. The code for this is [here](https://github.com/GitGuardian/ggshield/pull/503/files#diff-5ae06e2f9def4f580d16c631b45c2b6e1fbc9713af8b611d7abd7d67a9ea01d9R266): we only add the Scannable to the chunk if its content is not empty.